packages/yoastseo/src/scoring/assessments/readability/SentenceBeginningsAssessment.js

```diff
@@ -36,127 +35,123 @@ export default class SentenceBeginningsAssessment extends Assessment {
 	/**
 	 * Counts and groups the number too often used sentence beginnings and determines the lowest count within that group.
 	 *
-	 * @param {array} sentenceBeginnings The array containing the objects containing the beginning words and counts.
+	 * @param {SentenceBeginning[]} sentenceBeginnings The array containing the objects containing the beginning words and counts.
```

Comment:
Still learning with using Typescript, but I noticed that it's complaining that it cannot find `SentenceBeginning` when I run `tsc`. One way to solve this is by importing the `@typedef` which I learned from the `frontend-ai` package.
Something like:
```
/**
 * @typedef {import("../../../languageProcessing/researches/getSentenceBeginnings").SentenceBeginning } SentenceBeginning
 */
```

Reply:
Thanks a lot! That's super helpful. I've added this type definition. Then got annoyed we didn't have that in a lot of other places either, so I've added them in a lot of places in this PR 😅